ui/components/app/detected-token/detected-token-values/index.scss

```diff
@@ -1,6 +1,6 @@
 .detected-token-values {
   &__checkbox {
     position: absolute;
-    right: 12px;
+    right: -32px;
```

Comment:
Commented a fix in this PR https://github.com/MetaMask/metamask-extension/pull/14274#pullrequestreview-928193840